lms/templates/base.html.jinja2

```diff
@@ -40,9 +40,6 @@
 
       {% if context.rpc_server_config is defined %}
         <script type="application/json" class="js-rpc-server-config">{{ context.rpc_server_config|tojson }}</script>
-        {% for url in asset_urls("postmessage_json_rpc_server_js") %}
```

Comment:
We're confident that removing this doesn't break other parts of the app, e.g. content selection?

Reply:
not 100% sure, but seems to work

👀